05-java-ios/TextEndOfLineCharactersExample.java

```diff
@@ -0,0 +1,27 @@
+import java.io.*;
+import java.nio.charset.StandardCharsets;
+
+class TextEndOfLineCharactersExample {
```

Comment:
All the other files have a description that explains what the code does. Would it be possible to do this here as well?